app/views/permits/_section_one_form.html.erb

```diff
@@ -0,0 +1,53 @@
+<!-- t.boolean :sec1_q1
+    t.boolean :sec1_q2
+    t.boolean :sec1_q3
+    t.boolean :sec1_q4
+    t.boolean :sec1_q5
+    t.string :sec1_q6
+    t.boolean :sec1_q7
+    t.string :sec1_q8
+    t.string :sec1_q9
+    t.string :sec1_q10
+    t.boolean :sec1_q11
+    t.string :sec1_q12
+    t.string :sec1_q13
+    t.boolean :sec1_q14
+    t.string :firewatch_sign_start -->
+
+
+<h2>Section 1 - Hot Work Checklist</h2>
+<table>
+<tr>
+  <th>Question</th>
+  <th>(Y)</th>
+  <th>(N)</th>
+  <th>(N/A)</th>
+</tr>
+<tr>
+<td><%= label :sec1_q1, "I have completed pre-task planning for this work?" %></td>
+<td><%= f.radio_button :sec1_q1, true %></td>
+<td><%= f.radio_button :sec1_q1, false %></td>
```

Comment:
How are true and false mapped to Yes, No or N/A which are the enumerize options for these questions?